### PR TITLE
Add small test data to `rnafusion`

### PR DESCRIPTION
**Pull request by atrigila**
Related to: https://github.com/nf-core/rnafusion/issues/571
Related to: https://github.com/nf-core/rnafusion/issues/556

### CODE REVIEW

File: testdata/reference/arriba/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### `/arriba/*`
+
+Full reference files obtained from running the `nf-core/rnafusion` pipeline with the `--build_reference` and `--arriba` flag.
```
Comment (nschcolnicov) [inline testdata/reference/arriba/README.md:3]: Please share the full command and revision used to generate these files

File: testdata/reference/ensembl/README.md [hunk 2]
```diff
@@ -1 +1,9 @@
-This directory contains ensembl fasta and gtf files. 
+### `Homo_sapiens.GRCh38.102.genepred`
+### `Homo_sapiens.GRCh38.102.refflat`
+
+Full files obtained from running the `nf-core/rnafusion` pipeline with the `--build_reference` flag.
```
Comment (nschcolnicov) [inline testdata/reference/ensembl/README.md:4]: Also here, please share full command and revision used

File: testdata/reference/gffread/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+### `Homo_sapiens.GRCh38.102_chr4_subset.gff3`
```
Comment (nschcolnicov) [inline testdata/reference/gffread/README.md:1]: Also here add command and revision

File: testdata/reference/hgnc/README.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+### `hgnc_complete_set.txt`
```
Comment (nschcolnicov) [inline testdata/reference/hgnc/README.md:1]: Also here add command and revision

File: testdata/reference/rrnatranscripts/README.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+### `Homo_sapiens.GRCh38.102_rrna_intervals.gtf`
```
Comment (nschcolnicov) [inline testdata/reference/rrnatranscripts/README.md:1]: Also here add command and revision

**Review by nschcolnicov — APPROVED** [review]
Just corrections to the readme so that we can ensure full reproducibility of the files

File: testdata/reference/ensembl/README.md [hunk 6]
```diff
@@ -1 +1,9 @@
...
+
+### `Homo_sapiens.GRCh38.102.chr4.gtf`
+### `Homo_sapiens.GRCh38.102.chr4.1700000-54900000fa`
```
Comment (alanmmobbs93) [inline testdata/reference/ensembl/README.md:7]: is this name ok or should it be `.fa`?

**Pull request merged**